Orange/widgets/visualize/owscatterplot.py

```diff
@@ -380,8 +380,14 @@ def update_graph(self, attributes=None, reset_view=True, **_):
         self.graph.zoomStack = []
         if attributes and len(attributes) == 2:
             self.attr_x, self.attr_y = attributes
+        X = BasicStats(self.data[:, self.attr_x], self.attr_x)
+        Y = BasicStats(self.data[:, self.attr_y], self.attr_y)
```

Comment:
Actually, looking at `BasicStats` implementation, I think you might be right. In that case, code as-is is ok, I guess.
@lanzagar what do you think?